src/fshaper/fshaper.sh

```diff
@@ -3,10 +3,8 @@
 # Copyright (C) 2014-2024 Firejail Authors
 # License GPL v2
 
-TCFILE=""
-if [ -x "$(PATH=/usr/sbin:/sbin:/run/current-system/sw/bin command -v tc)" ]; then
-	TCFILE="$(PATH=/usr/sbin:/sbin:/run/current-system/sw/bin command -v tc)"
-else
+TCFILE="$(PATH=/usr/sbin:/sbin:/run/current-system/sw/bin command -v tc >/dev/null)"
+if [ ! -x "$TCFILE" ]; then
```

Comment:
```suggestion
if [ -z "$TCFILE" ]; then
```

Reply:
Not my best argh. Should be fixed now, thanks!